OpenRA.Mods.Common/Warheads/ChangeOwnerWarhead.cs

```diff
@@ -23,6 +23,10 @@ public class ChangeOwnerWarhead : Warhead
 
 		public readonly WDist Range = WDist.FromCells(1);
 
+		[TranslationReference(optional: true)]
+		[Desc("Text notification the victim will see when the unit is permanently converted.")]
+		public readonly string UnitStolenTextNotification = null;
```

Comment:
Are you actually using this downstream? If not IMO better to leave this out and take just the plumbing.